src/musig.h

```diff
@@ -40,8 +40,7 @@ std::optional<CPubKey> MuSig2AggregatePubkeys(const std::vector<CPubKey>& pubkey
 class MuSig2SecNonce
 {
 private:
-    //! The actual secnonce itself
-    secure_unique_ptr<secp256k1_musig_secnonce> m_nonce;
+    std::unique_ptr<MuSig2SecNonceImpl> m_impl;
```

Comment:
in commit 5e90a3c747ac2d44f9f52a3eeffbbe391f837bd8: seems like this change should be part of the earlier commit 01f639835783aa10d4c4b63da477c0300bc8495f, which currently doesn't compile
```
...
In file included from /home/thestack/bitcoin_prrev/pr29675/src/musig.cpp:5:
/home/thestack/bitcoin_prrev/pr29675/src/musig.h:44:5: error: no template named 'secure_unique_ptr'
    secure_unique_ptr<secp256k1_musig_secnonce> m_nonce;

/home/thestack/bitcoin_prrev/pr29675/src/musig.cpp:74:36: error: member initializer 'm_impl' does not name a non-static data member or base class
MuSig2SecNonce::MuSig2SecNonce() : m_impl{std::make_unique<MuSig2SecNonceImpl>()} {}
...
```
(not sure why the test-each-commit CI job doesn't detect this though?)

Reply:
@theStack it only runs the most recent N (6?) commits.

Reply:
Fixed